DentOS_Framework/DentOsTestbed/src/dent_os_testbed/constants.py

```diff
@@ -71,6 +72,7 @@
 }
 
 PYTEST_SUITE_GROUPS = {
+    "suite_group_bridging":["suite_bridging"],
```

Comment:
Here you should have:
    "suite_group_functional": ["suite_functional_bridging"],